source/Octopus.Manager.Tentacle/Octopus.Manager.Tentacle.csproj

```diff
@@ -119,7 +119,7 @@
     <PackageReference Include="MaterialDesignColors" Version="1.2.0" />
     <PackageReference Include="MaterialDesignThemes" Version="2.5.0.1205" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
-    <PackageReference Include="Microsoft.NETCore.Platforms" Version="1.1.0" />
+    <PackageReference Include="Microsoft.NETCore.Platforms" Version="2.2.0" />
```

Comment:
THis came with a update to Halibut

Reply:
Do we want this to just be a transient dependency?

Reply:
Yeh I can see if it is happy with that